lk2nd/device/dts/include/dt-bindings/gpio.h

```diff
@@ -0,0 +1,33 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+#ifndef DT_BINDINGS_GPIO_H
+#define DT_BINDINGS_GPIO_H
+
+/*
+ * Macro to set a phandle like 'GIO\x00'.
+ * Implementation will derive driver ID from this
+ */
+#define GPIO_DEV_PHANDLE(dev) (0x47494f00 | ((dev) & 0xff))
+
+#define GPIOL_DEVICE_TLMM	1
+#define GPIOL_DEVICE_PMIC	2
+#define GPIOL_DEVICE_PMIC_PON	3
+
+/* GPIO configuration flags */
+#define BIT(x) (1 << (x))
+
+#define GPIO_ACTIVE_HIGH	0
+#define GPIO_ACTIVE_LOW		BIT(24)
```

Comment:
I would probably try to reuse the official bits if it's not too much effort (but we can add additional ones if needed), simply to avoid some confusion.